internal/fields/validate.go

```diff
@@ -27,13 +27,20 @@ import (
 	"github.com/elastic/elastic-package/internal/multierror"
 	"github.com/elastic/elastic-package/internal/packages"
 	"github.com/elastic/elastic-package/internal/packages/buildmanifest"
+	"github.com/elastic/package-spec/v2/code/go/pkg/specerrors"
 )
 
+// EPF - Elastic Package Fields [validation]
+const ArrayOfObjectsErrorCode = "EPF00001"
```

Comment:
@mrodm do we have a place where we are already defining Elastic Package errors?

Reply:
Currently, there is no definition of errors in Elastic Package. Until now, it was just getting the errors from package-spec validate method and return the errors filtered (if any).

Should this be added into a new `internal/errors` package ? Or maybe another naming to not have collisions with standard errors package (e.g. validationerrors? I don't like neither too much this).

Reply:
Removed by now.